src/Pages/Form/Components/UnderLyingDiseaseSection.tsx

```diff
@@ -0,0 +1,121 @@
+import {
+  makeStyles,
+  FormGroup,
+  FormControlLabel,
+  Checkbox,
+  CheckboxProps,
+  withStyles,
+  TextField,
+} from "@material-ui/core";
```

Comment:
Please extract all imports to default imports(eg import Box from '@material-ui/core/Box'). Because importing object is importing all of component in @material-ui/core.